src/Hedgehog/Gen.fs

```diff
@@ -221,6 +252,7 @@ module Gen =
     /// is halved. When the size gets to one or less, selections are no longer made
     /// from the recursive list.
     /// <i>The first argument (i.e. the non-recursive input list) must be non-empty.</i>
+    [<CompiledName("ChoiceRecursive")>]
```

Comment:
In the Haskell version I actually split this in to two separate combinators [choice](http://hackage.haskell.org/package/hedgehog-0.4.1/docs/Hedgehog-Gen.html#v:choice) + [recursive](http://hackage.haskell.org/package/hedgehog-0.4.1/docs/Hedgehog-Gen.html#v:recursive) which you combine together, so I think this is fine for now.